assayist/processor/base.py

```diff
@@ -0,0 +1,121 @@
+# SPDX-License-Identifier: GPL-3.0+
+
+from abc import ABC, abstractmethod
+import json
+import neomodel
```

Comment:
It's best practice to have the following format (excluding comments):
```python
# standard library
from abc import ABC, abstractmethod
import json
import os

# third-party
import neomodel

# local imports
from assayist.common.models import content, source
from assayist.processor.configuration import config
from assayist.processor.logging import log
```

Reply:
Will fix.